### PR TITLE
feat(Clause): show red border on parse error

### PR DESCRIPTION
**Pull request by DianaLease**
Signed-off-by: Diana Lease <dianarlease@gmail.com>

### Changes

- feat(Clause): show red border on parse error
- unrelated change - updates snapshot tests from previous changes
- updates story to print out parse result or error in storybook actions tab

### Screenshots or Video
Screen Shot 2020-08-12 at 5 49 47 PM

### CODE REVIEW

File: packages/storybook/src/stories/4-ContractEditor.stories.js [hunk 1]
```diff
@@ -136,52 +136,58 @@ This is text. This is *italic* text. This is **bold** text. This is a [link](htt
...
+      const ciceroClause = new Clause(newReduxState[TEMPLATE_NAME]);
+      ciceroClause.parse(text)
+      const parseResult = ciceroClause.getData();
+      action('Clause -> Parse: ')({
+        clause: TEMPLATE_NAME,
+        parseResult,
+      });
+      return Promise.resolve(true);
+
+      /* XXX What do we do with this? - JS
```
Comment (dselman) [inline packages/storybook/src/stories/4-ContractEditor.stories.js:160]: We need to finish this...

File: packages/ui-contract-editor/src/lib/ContractEditor/plugins/withClauses.js [hunk 2]
```diff
@@ -75,12 +75,17 @@ const withClauses = (editor, withClausesProps) => {
...
-      const debouncedOnClauseUpdated = _.debounce(onClauseUpdated, 1000, { maxWait: 10000 });
       const [clauseNode] = Editor.nodes(editor, { match: n => n.type === CLAUSE });
-      debouncedOnClauseUpdated(clauseNode[0]);
+      onClauseUpdated(clauseNode[0]).then(success => {
+        if (success) {
+          Transforms.setNodes(editor, { error: false }, { at: clauseNode[1] });
+        } else {
+          Transforms.setNodes(editor, { error: true }, { at: clauseNode[1] });
+        }
+      });
```
Comment (irmerk) [inline packages/ui-contract-editor/src/lib/ContractEditor/plugins/withClauses.js:87]: Why are we removing debouncing here?

Comment (irmerk) [reply packages/ui-contract-editor/src/lib/ContractEditor/plugins/withClauses.js:87]: Note I think this originated here: https://github.com/accordproject/cicero-ui/pull/189

Comment (DianaLease) [reply packages/ui-contract-editor/src/lib/ContractEditor/plugins/withClauses.js:87]: I think the parent app should instead decide if to debounce and for how long, so the user of the component has full control. I'll add it to the storybook demo though as an example 👍

**Pull request merged**